api/interservice/authz/v2/project.proto

```diff
@@ -2,7 +2,6 @@ syntax = "proto3";
 
 import "validate/validate.proto";
 import "api/interservice/authz/v2/type.proto";
-import "api/interservice/event/event.proto";
```

Comment:
This is unrelated, I was just tired of seeing the warning generated during proto compilation from this.